### DISCUSSION TITLE
Can renovate work with git repository directly?

### DISCUSSION THREAD

**Discussion by AlexeyTsvetkov** [renovatebot/renovate#29675] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Other

### Please tell us more about your question or problem

I can't understand from the docs if it's possible to create update branches for raw git repositories (i.e. not a managed service like Github, Github Bitbucket, etc)? The problem is that the git hosting we're using is JetBrains Space, which does not seem to be supported. 
So I wonder if it's possible for renovate to work with git directly? (i.e. create and push branches via git, not via a hosting service)
If not, is it possible to contribute the support for JB Space?

**Comment by rarkins** [comment, marked as answer]
No, not yet supported. It would need an enhancement to support a config option like platform=local-git. platform=local exists but does not create branches.

**Reply by AlexeyTsvetkov** [reply]
Thanks for the clarification! Do you have plans to support local git repos?

**Reply by rarkins** [reply]
No, so you're welcome to submit a PR if you'd like to see it
